MapboxVision/src/main/java/com/mapbox/vision/sensors/SensorsManager.kt

```diff
@@ -135,21 +135,22 @@ internal class SensorsManager(application: Application) : SensorEventListener {
                             (value - gravity[index]) / SensorManager.GRAVITY_EARTH
                         }
                         .toFloatArray(),
-                    heading = heading.toFloat()
+                    heading = headingGeomagnetic.toFloat(),
```

Comment:
Rename field itself to headingGeomagnetic?

Reply:
Plus - should we pass trueHeading here, not magnetic?

Reply:
it's a heading and type is received in `attitude` field. 
I get that without geomagnetic sensor `SensorManager` doesn't work so we should always have `headingGeomagnetic `

Reply:
Ok